### ISSUE TITLE
Can't generate a new ELBfile

### ISSUE THREAD

**Issue by rramsden** [winebarrel/kelbim#3]
The following command doesn't seem to work for me:

```
kelbim -e -o ELBfile
```

Stacktrace:

```
/Users/richardramsden/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/kelbim-0.2.8/lib/kelbim/ext/elb-load-balancer-ext.rb:6:in `<class:LoadBalancer>': undefined method `attribute' for AWS::ELB::LoadBalancer:Class (NoMethodError)
        from /Users/richardramsden/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/kelbim-0.2.8/lib/kelbim/ext/elb-load-balancer-ext.rb:5:in `<class:ELB>'
        from /Users/richardramsden/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/kelbim-0.2.8/lib/kelbim/ext/elb-load-balancer-ext.rb:4:in `<module:AWS>'
        from /Users/richardramsden/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/kelbim-0.2.8/lib/kelbim/ext/elb-load-balancer-ext.rb:3:in `<top (required)>'
        from /Users/richardramsden/.rbenv/versions/2.1.5/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:73:in `require'
        from /Users/richardramsden/.rbenv/versions/2.1.5/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:73:in `require'
        from /Users/richardramsden/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/kelbim-0.2.8/lib/kelbim/exporter.rb:1:in `<top (required)>'
        from /Users/richardramsden/.rbenv/versions/2.1.5/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:73:in `require'
        from /Users/richardramsden/.rbenv/versions/2.1.5/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:73:in `require'
        from /Users/richardramsden/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/kelbim-0.2.8/lib/kelbim/client.rb:3:in `<top (required)>'
        from /Users/richardramsden/.rbenv/versions/2.1.5/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:73:in `require'
        from /Users/richardramsden/.rbenv/versions/2.1.5/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:73:in `require'
        from /Users/richardramsden/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/kelbim-0.2.8/lib/kelbim.rb:2:in `<top (required)>'
        from /Users/richardramsden/.rbenv/versions/2.1.5/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:73:in `require'
        from /Users/richardramsden/.rbenv/versions/2.1.5/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:73:in `require'
        from /Users/richardramsden/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/kelbim-0.2.8/bin/kelbim:6:in `<top (required)>'
        from /Users/richardramsden/.rbenv/versions/2.1.5/bin/kelbim:23:in `load'
        from /Users/richardramsden/.rbenv/versions/2.1.5/bin/kelbim:23:in `<main>'
```

Gemfile:

```
source "https://rubygems.org"

gem 'kelbim'
```

**Comment by winebarrel** [comment]
Thank you for your report :-)

**Comment by winebarrel** [comment]
This issue has been fixed in v0.2.9.
I'm sorry I'm late.
